src/pages/MyPage/components/BookMark/index.tsx

```diff
@@ -0,0 +1,48 @@
+import filledStar from 'assets/svg/mypage/star-filled.svg';
+import defaultImage from 'assets/images/search/default-image.png';
+import useScraps from 'pages/MyPage/hooks/useScraps';
+import useObserver from 'pages/MyPage/hooks/useObeserver';
+import notExist from 'assets/svg/mypage/not-exist.svg';
+import styles from './BookMark.module.scss';
+
+export default function BookMark() {
+  const {
+    scraps, isLoading, fetchNextPage, total,
+  } = useScraps();
+  const { target: bottom } = useObserver(fetchNextPage);
+  return (
+    <div className={styles.bookmarks}>
+      {!isLoading && scraps && (
+        <>
+          <span className={styles.bookmarks__total}>{`총 ${total}개의 음식점`}</span>
+          {scraps.map((scrap) => (
+            <div className={styles.bookmark} key={scrap.scrapId}>
+              <div className={styles.bookmark__detail}>
+                <span className={styles['bookmark__detail--name']}>{scrap.name}</span>
+                <span className={styles['bookmark__detail--type']}>{scrap.category}</span>
+              </div>
+              <div className={styles['bookmark__star-rate']}>
+                <img src={filledStar} alt="satr-rate" className={styles['bookmark__star-rate--image']} />
+                <span className={styles['bookmark__star-rate--rate']}>{parseFloat((scrap.totalRating / scrap.ratingCount).toString()).toFixed(1)}</span>
```

Comment:
둘 다 number형이니 나누고 fixed만 하면 안되려나요??
```suggestion
                <span className={styles['bookmark__star-rate--rate']}>{(scrap.totalRating / scrap.ratingCount).toFixed(1)}</span>
```

Reply:
엇 그렇네요 수정완료했습니다!